src/codeFly/dispatcher/LoginHandler.java

```diff
@@ -0,0 +1,82 @@
+package codeFly.dispatcher;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import codeFly.CodeFly;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.ArrayList;
+
+/**
+ * A place holder for module that have not been implemented.
+ */
+public class LoginHandler implements HttpHandler{
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        // Fetch request method
+    	
+    	String methodType = exchange.getRequestMethod();
+    	
+    	// GET method
+    	if(methodType.equals("GET")) {
+    		  // FIXME: request should provide the question number and server should return the proper question description
+            File loginPage = new File(CodeFly.ROOT_DIR + "frontEnd/login.html");
+            // response with a success response
+            exchange.sendResponseHeaders(200, loginPage.length());
+            OutputStream os = exchange.getResponseBody();
+            Files.copy(loginPage.toPath(), os);
+            os.close();
+            CodeFly.logger.info(String.format("Sending %s to client: %S",
+                    loginPage.getName(), exchange.getRemoteAddress()));
+    	}
+    	
+    	// POST method
+    	else {
+    		//Parse the request
+            BufferedReader reader = new BufferedReader(new InputStreamReader(exchange.getRequestBody()));
+            String line;
+            ArrayList<String> loginInfo = new ArrayList<String>();

```

Comment:
There should only be two fields in the `POST` request: a user name and a password. There's no need to use an `ArrayList` to store just two strings.